.github/workflows/node.js-tests.yml

```diff
@@ -117,6 +117,43 @@ jobs:
       - name: Run Tests
         run: pnpm test
 
+  # DONT REMOVE THIS JOB, you dont know what it does
```

Comment:
Can I speculate? This tests if it's possible to build the client in a non-flaky workflow. That means we can test it in CI and get alerted to build failures before they're in main.

Reply:
Hmm...that's like the one job the CI has. 😄